src/table/bridge_land.h

```diff
@@ -740,8 +740,12 @@ static const std::span<const std::span<const PalSpriteID>> _bridge_sprite_table[
  * @param nrl description of the rail bridge in query tool
  * @param nrd description of the road bridge in query tool
  */
-#define MBR(y, mnl, mxl, p, mxs, spr, plt, dsc, nrl, nrd) \
-	{TimerGameCalendar::Year{y}, mnl, mxl, p, mxs, spr, plt, dsc, { nrl, nrd }, {}, 0}
+#define MBR(y, mnl, mxl, p, mxs, spr, plt, dsc, nrl, nrd, pillars) \
+	{TimerGameCalendar::Year{y}, mnl, mxl, p, mxs, spr, plt, dsc, { nrl, nrd }, {}, 0, 0, pillars}
+
+#define ALL_PILLARS  { 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F, 0x0F }
+#define SUSP_PILLARS { 0x03, 0x06, 0x0C, 0x09, 0x0C, 0x09, 0x03, 0x06, 0x0F, 0x0F, 0x00, 0x00 }
+#define CANT_PILLARS { 0x00, 0x00, 0x0C, 0x09, 0x0C, 0x09, 0x0C, 0x09, 0x0C, 0x09, 0x0C, 0x09 }
```

Comment:
Defines: declare these as `static constexpr BridgePiecePillarFlags` arrays or `std::initializer_list` instead.
Magic numbers: please use the defined enum names instead.